runtime/lua/vim/lsp/log.lua

```diff
@@ -132,19 +132,18 @@ local function create_logger(level, levelnr)
     end
     local info = debug.getinfo(2, 'Sl')
     local header = string.format(
-      '[%s][%s] ...%s:%s',
+      '[%s][%s] %s:%s',
       level,
       os.date(log_date_format),
-      info.short_src:sub(-16),
+      info.short_src,
       info.currentline
     )
-    local parts = { header }
-    for i = 1, argc do
-      local arg = select(i, ...)
-      table.insert(parts, arg == nil and 'nil' or format_func(arg))
```

Comment:
You should still use `format_func`. That's a [user-configurable function](https://neovim.io/doc/user/lsp.html#vim.lsp.log.set_format_func()) that handles how these logs are printed.